zulip_bots/zulip_bots/test_lib.py

```diff
@@ -25,7 +26,7 @@
 
 class StubBotHandler:
     def __init__(self) -> None:
-        self.storage = SimpleStorage()
+        self.storage = SimpleStorage()  # type: CommonStorage
```

Comment:
This looks good, though I'm not sure if we need a separate protocol for this or if this will be a common implementation.
I would put this refactor into a separate commit.
We should be able to use an inline type annotation.

Reply:
The only way to provide a static type to storage is by using a Protocol.
Otherwise making it an object is fine as well, but I think it is a better way to go.

Reply:
Pushed in a separate commit.